scripts/simulate_agents.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+import logging
+import os
+from multiprocessing import Process
+from time import sleep
+
+import wazuh_testing.tools.agent_simulator as ag
+from wazuh_testing import TCP
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def run_agents(agents_number=1, manager_address='localhost', protocol=TCP, agent_version='v4.0.0',
+               agent_os='debian8', eps=1000, run_duration=20, active_modules=None, modules_eps=None):
+    """Run a batch of agents connected to a manager with the same parameters.
+
+    Args:
+        agents_number (int): Number of agents to run.
+        manager_address (str): Manager address (hostname or IP).
+        protocol (str): Communication protocol.
+        agent_version (str): Agents version p.e: v4.0.0
+        agent_os (str): Agents os, some examples: debian8, ubuntu18.04, mojave...
+        eps (int): Total events per second sent by each agent to the manager.
+        run_duration (int): duration in seconds of the agents run.
```

Comment:
Please change this, it could be something like:

```suggestion
        run_duration (int): Agent life time.
```

Also the name of the variable `run_duration` sounds rather strange.